src/Contracts/DataModels.ts

```diff
@@ -22,6 +22,7 @@ export interface DatabaseAccountExtendedProperties {
   enableAnalyticalStorage?: boolean;
   isVirtualNetworkFilterEnabled?: boolean;
   ipRules?: IpRule[];
+  privateEndpointConnections?: unknown[];
```

Comment:
>unknown [](http://example.com/codeflow?start=31&length=7)

Can you check ?

Reply:
I referred to generatedArmClient that we have and that also uses `unknown[]` data type 
https://github.com/Azure/cosmos-explorer/blob/0201e6ff92f218085ffcecec180305256d8a0b12/src/Utils/arm/generatedClients/cosmos/types.ts#L424

@southpolesteve do you know if this property can be strongly typed?

Reply:
The generator has some edge cases it does not handle so it will bail out to `unknown `.  File an issue and I'll look into it.

Its ok to edit the generated type here. Write a comment saying why you edited it. I don't think we should get in the habit of editing this file but exceptions will exist.

Reply:
Is it okay to check this in and have a follow up to change this to a strongly typed definition instead? That would require some testing (for which I'll need Srinath's help) and would delay the fix.

Reply:
Also created an issue here for tracking
https://github.com/Azure/cosmos-explorer/issues/841

Reply:
Yes. If you don't need the contents of the `unknown` its fine to leave this as is.